tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -806,3 +807,41 @@ func saveRuleBundle(cmd *cobra.Command, args []string) {
 
 	cmd.Println(res)
 }
+
+// NewSwitchWitnessCommand placement rules subcommand
+func NewSwitchWitnessCommand() *cobra.Command {
```

Comment:
Why not use the config command directly?

Reply:
Removed, thanks!